qcodes_contrib_drivers/tests/QDevil/readme.md

```diff
@@ -27,3 +27,4 @@ Static types:
     $ source venv/bin/activate
     $ pip install --upgrade pip
     $ pip install qcodes pytest pyvisa-sim pyvisa-py mypy
+    $ pip install -e .
```

Comment:
The aim of how our tests are set up hopefully it should be enough to do `pip install -e .[test]` if any deps are missing we should add them there

Reply:
Ahh, I did not know that you could write `pip install -e .[test]`